### ISSUE TITLE
extraKeys can not add 'Enter' key

### ISSUE THREAD

**Issue by kongchenglc** [RIP21/react-simplemde-editor#183]
```js
import SimpleMdeReact from 'react-simplemde-editor'

...

    <SimpleMdeReact
      value={value}
      onChange={onChange}
      options={options}
      getCodemirrorInstance={getCmInstanceCallback}
      getLineAndCursor={getLineAndCursorCallback}
      events={eventsHandlers}
      extraKeys={{
            Enter: ()=>{} // it is useless
      }}
    />
...

```

**Comment by coltenkrauter** [comment]
I experienced the same issue. I was able to work around it via events. 

```
 <SimpleMDE
     events={{
         keyHandled: (editor, event) => {
             if (event == 'Enter') {
                 console.log('Enter pressed');
             }
         },
     }}
 />
```

**Comment by ChesterZengJian** [comment]
I guess 'codemmirror.getoption ("extraKeys")' overrides the 'Enter' key in the extraKeys

![image](https://user-images.githubusercontent.com/22461405/189274911-e6ef3167-7437-43b3-9416-5e5ceedcc2e6.png)

**Comment by RIP21** [comment]
Almost a year passed lol.
Fixed this one in `5.1.1` already.

**Comment by coltenkrauter** [comment]
Thanks!
